src/msg/async/EventKqueue.cc

```diff
@@ -199,13 +199,14 @@ int KqueueDriver::resize_events(int newsize)
   ldout(cct,30) << __func__ << " kqfd = " << kqfd << "newsize = " << newsize 
                 << dendl;
   if(newsize > sav_max) {
-    sav_events = (struct SaveEvent*)realloc( sav_events, 
-                    sizeof(struct SaveEvent)*newsize);
-    if (!sav_events) {
+    void *_realloc = NULL;
+    if ((_realloc = realloc(sav_events, izeof(struct SaveEvent)*newsize)) == NULL) {
```

Comment:
s/izeof/sizeof/, could use "if with initialization" introduced by C++17, like
```c++
if (void* new_events = realloc(sav_events, sizeof(struct SaveEvent)*newsize)); !new_events) {
  lderr(cct) << __func__ << " unable to realloc memory: "
                               << cpp_strerror(errno) << dendl;
  free(sav_events);
  return -ENOMEM;
} else {
  sav_events = new_events;
}
```

Reply:
Strange, it compiled here fine.